my-webxr-app/src/components/DataPoint.tsx

```diff
@@ -43,17 +46,37 @@ export default function DataPoint({
     setHoverCount(amount);
   };
 
+  // Display the point characteristics in console
```

Comment:
are these console.log() meant to be in the PR or are these just for debugging locally?

Reply:
They are for debugging locally! Also, if several points are selected together and the text is cluttered in the space, we can easily distinguish the different fields for each point if printed to the console. 

What are your thoughts?